lib/vtls/bearssl.c

```diff
@@ -474,7 +474,7 @@ static CURLcode bearssl_connect_step1(struct Curl_easy *data,
     hostname = snihost;
   }
 
-  if(!br_ssl_client_reset(&backend->ctx, hostname, 0))
+  if(!br_ssl_client_reset(&backend->ctx, hostname, 1))
```

Comment:
curl has its own session management. How does this affect the call to `br_ssl_engine_set_session_parameters` above?

Reply:
Passing 0 here causes BearSSL to disable session resumption, so it won't resume a session with parameters set earlier. Passing 1 instead will cause BearSSL to actually use those parameters.

This was just an oversight on my part; I don't think I actually tested session resumption, so I forgot to enable it.